### PR TITLE
DDP-6728 redesign colorectal page

### PR DESCRIPTION
**Pull request by aglushko-broad**

### CODE REVIEW

**Comment by alextanlan** [comment]
Can you please increase font-size for paragraph-text for the page sections (**except introduction and learn-more sections**)
like you did it for welcome page

```
:host ::ng-deep .static-page-paragraph-text {
    font-size: 1.5rem;
    line-height: 36px;
}
```

**Comment by alextanlan** [comment]
#minor
Links in 'learn-more' sections are bold on mockups
   Implementation:
image

  Mockups:
image

File: ddp-workspace/projects/ddp-pancan/src/app/components/app/app.component.ts [hunk 1]
```diff
@@ -16,7 +17,8 @@ export class AppComponent extends AppRedesignedBaseComponent {
     phone: string;
     email: string;
     isSplashPage: boolean;
-    private readonly splashPageUrls = ['colorectal', 'lms'];
+    isColorectalPage: boolean;
+    private readonly splashPageUrls = [AppRoutes.ColorectalPage, AppRoutes.LMS];
```
Comment (alextanlan) [inline ddp-workspace/projects/ddp-pancan/src/app/components/app/app.component.ts:21]: Nice!

**Comment by MocanaAtBroad** [comment]
We should try to fit the "how to participate" steps in the available horizontal space (Step 4 is wrapping below the other ones even though it appears there is plenty of space in fully open desktop browser):
Screen Shot 2021-08-09 at 1 35 30 PM

**Comment by aglushko-broad** [comment]
> Can you please increase font-size for paragraph-text for the page sections (**except introduction and learn-more sections**)
> like you did it for welcome page
> 
> ```
> :host ::ng-deep .static-page-paragraph-text {
>     font-size: 1.5rem;
>     line-height: 36px;
> }
> ```

done!

**Comment by aglushko-broad** [comment]
> #minor
> Links in 'learn-more' sections are bold on mockups
> Implementation:
> image
> 
> Mockups:
> image

done!

**Comment by aglushko-broad** [comment]
> We should try to fit the "how to participate" steps in the available horizontal space (Step 4 is wrapping below the other ones even though it appears there is plenty of space in fully open desktop browser):
> Screen Shot 2021-08-09 at 1 35 30 PM

done!

**Review by alextanlan — APPROVED** [review]

**Pull request merged**